app/controllers/assistance_controller.rb

```diff
@@ -1,18 +1,22 @@
 # Controller for the get assistance pages
 class AssistanceController < ApplicationController
   def housing_counselors
+    @AssistanceProps = {assetPaths: static_asset_paths}
```

Comment:
Just wondering why we're creating all of these instance variables. We get equivalent functionality by calling the `static_asset_paths` helper method inside the view:

```slim
// app/views/assistance/get_assistance.html.slim
== react_component 'GetAssistance', props: { assetPaths: static_asset_paths }
```

Reply:
For sure, we could totally do it that way too! The reason I changed it was because the implementation of just writing `assetPaths: static_asset_paths` wasn't working. My logic for creating the instance variables was that it felt cleaner and easier to read for folks who are new to ruby/react-on-rails. 

Totally happy to change it to your way if you think that is better 😄

Reply:
I usually prefer to have less code, but if you think it will help others then it may be worth keeping (there's always a balance between terseness and readability). One nit I do have is to use snake_case for variables coming from ruby-land, e.g. `@assistance_props`.